lib/guides-markdown-export.json

```diff
@@ -0,0 +1,30 @@
+{
+    "beginner": {
+      "beginners guide": {
```

Comment:
if you hyphenate these keys then they are much easier to type into a url instead of using a space. you can even drop the `decodeURI` call.